setup-nightly.py

```diff
@@ -1,34 +1,35 @@
 import setuptools
 
-with open("README-nightly.md", "r") as fh:
+with open('README-nightly.md', 'r') as fh:
```

Comment:
How is this related?

Reply:
As a side effect to adding the `author` label on our manifest files, we've also replaced double quotes with single ones